app/RouteHandlers/AddDonationHandler.php

```diff
@@ -93,6 +93,15 @@ private function newHttpResponse( AddDonationResponse $responseModel ): Response
 					)
 				);
 				break;
+			case PaymentType::SOFORT:
+				$httpResponse = $this->app->redirect(
+					$this->ffFactory->newSofortUrlGeneratorForDonations()->generateUrl(
+						$responseModel->getDonation()->getId(),
```

Comment:
For Till it was more important to have a `WQ-Code` (transfer code) here, like in the bank transfer payment. That has to be generated when the donation is persisted and then read here from the payment data. Maybe add a TODO or FIXME here, since the transfer code is not generated yet?